### ISSUE TITLE
No -webkit- prefix for transform

### ISSUE THREAD

**Issue by vedraan** [rofrischmann/inline-style-prefixer#26]
I'm using https://github.com/callemall/material-ui which is currently using inline-style-prefixer@0.5.1

I noticed that support for transform is missing on Android native browser 4.*, 
It needs to be prefixed with `-webkit-` according to caniuse:
http://caniuse.com/#feat=transforms3d

I thought it might be an user agent problem so I also tried forcing prefixAll option which adds the `-webkit-` (as well as all other prefixes) to `transition` and `transform-origin`, but not to `transform` so the problem is not with detection.

**Comment by rofrischmann** [comment]
@vedraan Thanks for reporting. Seems strange because if I check the data (*lib/caniuseData.js*) it lists `transform: 4.4` which means transform needs `-webkit-` prefix for android up to version 4.4.

But let me check that as soon as possible.

**Comment by rofrischmann** [comment]
I figured out that the browser detection gets something like:
```javascript
{
   // Chrome version
   version: 30,
   osversion: 4.4
}
```
The prefixer uses the version to validate props so of course, as 30 > 4.4 it does not add those.
This is an issue unique to Android, because caniuse.com use the osversion instead of the built-in Chrome version (which makes sense). Starting with Android > 5 caniuse.com start to reference the Chrome version (44 right now). 

I fixed this by swapping the osversion and version for Android < 4.4 if the native one got detected.
